### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by whizdummy** [wolverine2k/Laravel-Backendless#2]
![error](https://cloud.githubusercontent.com/assets/12737833/14764447/6f96f712-09ea-11e6-9901-0f08f17b7fff.png)

**Comment by wolverine2k** [comment]
Yes, this is a known issue. For the time being, I have not uploaded the package in https://packagist.org/. I have been waiting for the backendless guys to do it since this package has dependencies towards it.

Feel free to upload this package but will not be of any help until the backendless sdk is also available as a package. In the meantime, please do manual installation of both the backendless sdk and this package.
